quicklogic/openfpga/tests/quicklogic_testsuite/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+add_custom_target(all_openfpga_k4_N8_tests_no_adder)
+add_custom_target(all_openfpga_k4_N8_tests_adder)
+
```

Comment:
A bunch of these tests should be **common** tests which are run on all architectures and under https://github.com/SymbiFlow/symbiflow-arch-defs/tree/master/tests directory.

Reply:
I added the board with the QuickLogic device to the relevant common tests.